.gitignore

```diff
@@ -18,3 +18,4 @@ Puppetfile.lock
 .*.sw?
 .yardoc/
 Guardfile
+Dockerfile.override
```

Comment:
this file is managed by our modulesync_config. Add it to https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.gitignore.erb or add it to the ```.sync.yml``` of this repo.